crates/next-custom-transforms/tests/fixture/debug-fn-name/anonymous-component/output.js

```diff
@@ -0,0 +1,14 @@
+const Component = ()=>{
+    useLayoutEffect({
+        "Component.useLayoutEffect": ()=>{}
+    }["Component.useLayoutEffect"]);
+    useEffect({
+        "Component.useEffect": ()=>{}
+    }["Component.useEffect"]);
+    const onClick = useCallback({
+        "Component.useCallback(onClick)": ()=>[]
+    }["Component.useCallback(onClick)"]);
+    const computed = useMemo({
+        "Component.useMemo(computed)": ()=>{}
```

Comment:
@kdy1 Actually, because Chrome parses the parenthesis as part of the URL when this is part of an error stack, we're better off using `[]`. 

`Component.useMemo[computed]`

Reply:
https://github.com/vercel/next.js/pull/68142